tests/gold_tests/h2/http2.test.py

```diff
@@ -79,9 +79,5 @@
 tr=Test.AddTestRun()
 tr.Processes.Default.Command='python3 h2bigclient.py -p {0}'.format(ts.Variables.ssl_port)
 tr.Processes.Default.ReturnCode=0
-# time delay as proxy.config.http.wait_for_cache could be broken
-tr.Processes.Default.StartBefore(server,ready=When.PortOpen(server.Variables.Port))
```

Comment:
Wait my bad... 

This line is still needed.. but it can not be `tr.Processes.Default.StartBefore(server)`